libraries/AP_HAL_ChibiOS/hwdef/SVehicle-E2/hwdef.dat

```diff
@@ -0,0 +1,344 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the S-Vehicle E2-plus AutoPilot hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_SVehicle-E2
+
+FLASH_RESERVE_START_KB 128
+
+# to be compatible with the px4 bootloader we need
+# to use a different RAM_MAP
+env USE_ALT_RAM_MAP 1
+
+# flash size
+FLASH_SIZE_KB 2048
+
+env OPTIMIZE -O2
```

Comment:
We tend to be leaving this out these days

Reply:
ok